### PR TITLE
SPRIN-79: Accordion FAQ Gift Cards test

### PR DESCRIPTION
**Pull request by vitalik1985o**
image

Finish Avtomation https://vitaliyosyp9282.atlassian.net/jira/software/projects/SPRIN/boards/3?selectedIssue=SPRIN-79

### CODE REVIEW

File: src/main/java/common/BasePage.java [hunk 1]
```diff
@@ -19,7 +19,7 @@ public class BasePage {
     protected static WebDriver driver;
     protected WebDriverWait wait;
 
-    protected BasePage(WebDriver driver) {
+    public BasePage(WebDriver driver) {
```
Comment (Goldbe54) [inline src/main/java/common/BasePage.java:22]: why `public`?

File: src/main/java/common/BasePage.java [hunk 2]
```diff
@@ -33,7 +33,7 @@ protected WebElement waitVisibilityOfElement(String locator) {
         }
     }
 
-    protected WebElement waitClickabilityOfElement(String locator) {
+    public WebElement waitClickabilityOfElement(String locator) {
```
Comment (Goldbe54) [inline src/main/java/common/BasePage.java:36]: why `public`?

File: src/main/resources/suites/defaultSuite.xml [hunk 3]
```diff
@@ -9,7 +9,11 @@
     <test name="All tests">
         <classes>
             <class name="tests.SignInValidationTest" > </class>
+
             <class name="tests.SignUpNewsletterTest" > </class>
+
+            <class name="tests.GiftCardsAccordingFaqTest" > </class>
+
```
Comment (Goldbe54) [inline src/main/resources/suites/defaultSuite.xml:16]:
```suggestion
            <class name="tests.SignInValidationTest" > </class>
            <class name="tests.SignUpNewsletterTest" > </class>
            <class name="tests.GiftCardsAccordingFaqTest" > </class>
```

**Review by Santakloun1 — APPROVED** [review]

File: src/test/java/tests/GiftCardsAccordingFaqTest.java [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        softAssert.assertTrue(giftCardsPage.getFrequentlyAskedQuestionsElement().isDisplayed(),
+                "FAQ section is not displayed on the page");
+
+        List<WebElement> faqPanels = giftCardsPage.getGiftCardsFaqPanels();
+        List<WebElement> faqExtendedContents = giftCardsPage.getGiftCardsFaqExtendedContents();
+
+        for (WebElement content : faqExtendedContents) {
+            softAssert.assertFalse(content.isDisplayed(), "FAQ panels are not collapsed");
+        }
+        for (int i = 0; i < faqPanels.size(); i++) {
```
Comment (Goldbe54) [inline src/test/java/tests/GiftCardsAccordingFaqTest.java:38]:
```suggestion
        }
        
        for (int i = 0; i < faqPanels.size(); i++) {
```

**Review by Goldbe54 — APPROVED** [review]

**Pull request merged**